R-packages/covidcast/inst/shapefiles/county/cb_2019_us_county_5m.cpg

```diff
@@ -0,0 +1 @@
+UTF-8
```

Comment:
Is this an extraneous output file?  If so we should probably mask it in a `.gitignore`

Reply:
Shapefiles are a bizarre collection of several files in one folder; I think both these files are probably necessary. (For example, the .prj defines the geographic projection used for the shapefile's coordinates)

Reply:
No, All these files would need to be used if we call the county shp file.  This hasn't happened yet, but I'm planning on using those to add Puerto Rico in a subsequent PR.